pkg/stub/handler.go

```diff
@@ -55,8 +55,26 @@ func (h *Handler) Handle(ctx types.Context, event types.Event) error {
 			return err
 		}
 
+		// get tokens for api access
+		if oneagent.Status.Tokens == "" || oneagent.Spec.Tokens != "" && oneagent.Status.Tokens != oneagent.Spec.Tokens {
```

Comment:
can we have a variable for each of the conditions?
eg *oneagent.Spec.Tokens != ""* is checked again in the next if. would improve readability a lot